### PR TITLE
Translate website to English (dev branch)

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
### Todo

https://github.com/DefactoSoftware/website/milestone/1

### Merged

- [x] CAPP Agile Learning #618 
- [x] Implementations #628 
- [x] Starterkit #620 
- [x] KPP #622 
- [x] Download productsheet #642 
- [x] References #635
- [x] Media #636
- [x] Contact #639 
- [x] Disable careers #629 
- [x] Services #627 
- [x] Hosting & Security #631 
- [x] Integrations #632
- [x] Testimonials #633
- [x] About us #634 
- [x] Localize & translate header #597 
- [x] Add English testimonials #598 
- [x] Demo request #608
- [x] English Agile testimonials #602 
- [x] Homepage #603 
- [x] Footer #604 
- [x] Popup #605
- [x] Thank you #607 
- [x] E-Learning page #612 
- [x] 404 #606
- [x] CAPP Page #610 
- [x] Search #626 
- [x] Blog #625 
- [x] 360 #623 
- [x] Products #616
- [x] Services sidebar #614 
- [x] CAPP LMS #610

### CODE REVIEW

**Comment by fatboypunk** [comment]
Are there pages we can drop for now? Maybe start with the homepage, contacts and capp agile learning? try to merge that and then move foreward? Or are we missing to much by then?

Either way I think it would be nice to keep this as a master branch for EN and have several PR's based on this branch so each PR is checkable.

**Comment by fatboypunk** [comment]
@sn3p you're oke with my suggestion?

**Comment by sn3p** [comment]
> @sn3p you're oke with my suggestion?

Yea might be a good idea. I suggested to wait for decent translations needed for the minimal deployable version.

**Comment by RonaldDijkstra** [comment]
Thanks guys

File: data/capp-agile-testimonials_en.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#   summary: true     # show summary if present, otherwise the full quote (optional)
+# } %>
+#
+#####################################################################################################
+
+- name: Peter van Eijl
+  company: Founder, Doppio Espresso
+  avatar: testimonials/peter_van_eijl.jpg
+  quote: >
+    <p>Translated</p>
```
Comment (fatboypunk) [inline data/capp-agile-testimonials_en.yml]: this should change I guess?

File: data/downloads.yml [hunk 2]
```diff
@@ -9,6 +9,8 @@
 ebook:
   nl: "/pdf/Handboek_Leren_en_laten_Leren.pdf"
   de: "/pdf/Handbuch_Abenteuer_E-Learning.pdf"
+  en: "/pdf/Handbook_Learn_and_let_learn.pdf"
 capp_productsheet:
   nl: "/pdf/Productsheet_CAPP.pdf"
   de: "/pdf/Produktblatt_CAPP.pdf"
+  en: "/pdf/Productsheet_CAPP.pdf" # REPLACE_LATER!
```
Comment (fatboypunk) [inline data/downloads.yml]: Can we make a milestone on github with these things as an issue? it's to easy to leave this in the comments and never again look at it, while a milestone with several issues can be completed and give a clear status of what has to be done

File: source/localizable/index.html.erb [hunk 3]
```diff
@@ -1,10 +1,12 @@
 ---
 title:
   nl: Developing People sinds 1987
+  en: Developing People sincs 1987
```
Comment (fatboypunk) [inline source/localizable/index.html.erb]: typo

File: data/capp-agile-testimonials_en.yml (hunk 1 above)
Comment (RonaldDijkstra) [inline data/capp-agile-testimonials_en.yml]: Yup, follows in a seperate PR. I just needed the file here.

File: data/downloads.yml (hunk 2 above)
Comment (RonaldDijkstra) [inline data/downloads.yml]: https://github.com/DefactoSoftware/website/milestone/1

File: data/capp-agile-testimonials_en.yml (hunk 1 above)
Comment (RonaldDijkstra) [inline data/capp-agile-testimonials_en.yml]: #602

File: source/localizable/index.html.erb (hunk 3 above)
Comment (RonaldDijkstra) [inline source/localizable/index.html.erb]: Fixed in #603

**Comment by RonaldDijkstra** [comment]
This is soft released 15 minutes ago on en.defacto.nl (waiting for the DNS to catch up).

When everything works, I'd like this branch to be reviewed and merged into master. 💃 

Thanks all :)

**Pull request merged**